welcome.md

```diff
@@ -0,0 +1,37 @@
+---
+

```

Comment:
Hey Mhairi I tried for quite some time to make this work with a more detailed What is UMA section and at the end of it decided it is best to remove it from the welcome page and have it in it's own section.